browser/components/CodeEditor.js

```diff
@@ -204,6 +208,29 @@ export default class CodeEditor extends React.Component {
     textarea.value = `${textarea.value.substr(0, textarea.selectionStart)}${imageMd}${textarea.value.substr(textarea.selectionEnd)}`
   }
 
+  handlePaste (editor, e) {
+    const dataTransferItem = e.clipboardData.items[0]
+    if (!dataTransferItem.type.match('image')) return
+
+    const blob = dataTransferItem.getAsFile()
+    let reader = new FileReader()
+    let base64data
+
+    reader.readAsDataURL(blob)
+    reader.onloadend = () => {
+      base64data = reader.result.replace(/^data:image\/png;base64,/, '')
```

Comment:
The meaning of this line
https://stackoverflow.com/questions/11511511/how-to-save-a-png-image-server-side-from-a-base64-data-string#comment36092057_11511552